lib/html/includes.js

```diff
@@ -640,6 +640,17 @@ var _MiniProfiler = (function() {
   };
 
   var bindDocumentEvents = function bindDocumentEvents() {
+    document.addEventListener("turbo:before-visit", e => {
```

Comment:
it looks like the other methods do not use anonymous blocks, they use functions defined. i.e.: onClickEvents, unbindDocumentEvents

Also, it looks like there is a check that `Turbolinks` is even relevant (by checking the constant defined in a gem. is there something similar you can do here?

Probably want to move this to the bottom of this method, right after Turbolinks section. That way the meat of this bindDocumentEvents method reads the same

Reply:
Thanks, @kbrock. 

I refactored the callbacks as functions and moved the code under Turbolinks logic.

`Turbo` doesn't implement `supported` like `Turbolinks`, so I am just checking that `Turbo` is available.

Doing the Turbo constant check avoid adding unnecessary event listeners, but it requires that the application exposes `Turbo` globally:

```
import { Turbo } from "@hotwired/turbo-rails"
window.Turbo = Turbo
```

What do you think? What is the best approach? I guess we should mention it in the documentation.